.travis.yml

```diff
@@ -33,6 +33,8 @@ matrix:
           # Test against LTS versions
         - php: 7.0
           env: SYMFONY_VERSION=2.8.*
+        - php: 7.1
+          env: SYMFONY_VERSION=4.0.*
```

Comment:
Write version 3.4 here instead. Then add another entry for 4.0 above the comment say "Test against LTS versions"